### DISCUSSION TITLE
Nuxt v3 integration example

### DISCUSSION THREAD

**Discussion by divine** [fingerprintjs/BotD#58] [Ideas]
Hello,

Is there any example to integrate with Nuxt v3? It's currently in RC and will be released soon https://v3.nuxtjs.org/

Runs on Clouds as well (CloudFlare/Vercel/Etc.) as well as middleware https://v3.nuxtjs.org/guide/features/server-routes#server-middleware.

Thanks!

**Comment by Romchela** [comment]
Hello,

Thanks for the idea. We will think about it in the future.
Now we're busy working on the release of BotD 1.0.
